successstories/index.html

```diff
@@ -0,0 +1,156 @@
+                            </div>
+                        </div>
+                        <div>
+                            <h5 class="name"><a href="#">{{mentee_name}}</a></h5>
+                            <p class="details">{{program_year}} Graduate</p>
+                        </div>
+    
+                    </div>
+                </div>
+                <div class="mentor col">
+                    <h5 class="mentor-details-title">Mentored by :</h5>
+                    <div class="row mentor-details">
+                        <div>
+                            <div class="mentor-profile-pic">
+                                <img class="img" src="{{mentor_img_url}}">
+                            </div>
+                        </div>
+                        <div>
+                            <p class="name"><a href="#">{{mentor_name}}</a></p>
+                            <p class="details">{{mentor_details}}</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {{/arr}}
+</script>
+
+<!-- Core -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.10.4/headroom.min.js"></script>
+<!-- Optional JS -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/14.0.3/nouislider.min.js"></script>
+<!--Mustache.js-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mustache.js/2.3.0/mustache.min.js"></script>
+<!-- Argon JS -->
+<script src="/assets/js/argon.js?v=1.0.1"></script>
+<!-- Script for static navbar and footer -->
+<script src="/assets/js/navigation.js"></script>
+<!-- External JS file for the JS functions-->
+<script src="functions.js"></script>
+
+<script>
+    const ss_data_url = 'https://script.googleusercontent.com/macros/echo?user_content_key=vYNRb_6wLpdAb7yohOGVZ6plAPRQYHj24PwHaRzyPeBAHv_IOO-iTocAcegU32lanTlNXAD5xprr1QcKF6AYOnasyELXO0mgm5_BxDlH2jW0nuo2oDemN9CCS2h10ox_1xSncGQajx_ryfhECjZEnGloHkTCAUi4CAqGZJpwR2I9IZqiKSN007f2pT1Z8Vo-bFbrfv4vEmQOrefG-1CINfwjcROloh7GA4MBynjXskzQS_1kBKld8w&lib=M5DrI9BMFrlzrW1XQTqlL3oZDs2x4KkTf';
+
+    async function getStories() {
+        const response_ss = await fetch(ss_data_url);
+        const data_ss = await response_ss.json();
+        const {Sheet1} = await data_ss;
+        Sheet1.forEach(element => {
+            const{ mentee_name, mentee_linkedin, programme_year, mentor_name, mentor_details_occupation, mentor_linkedin, highlight, story, achievements } = element;
+
+
+            //get from linkedin
+            let mentee_img_url = "https://www.w3schools.com/w3css/img_avatar3.png";
+            let mentor_img_url = "https://www.w3schools.com/w3css/img_avatar3.png";
```

Comment:
This should be in the google sheet

Reply:
Okay. And I should ask SX team to not to fill those columns right? because we have to fill them from Cloudinary?

Reply:
they need to add the linkedin username. Then we have to manually download them and upload it to cloudinary.

Reply:
and then store that cloudinary link manually on the google sheet?